docs/source/releasenotes.rst

```diff
@@ -19,6 +19,7 @@ Release notes
 
 - Library **RPA.Excel.Files** (:pr:`490`): Keyword examples updated to be more complete
   and Python examples have been added to all keywords.
+- Library **RPA.Tables** (:pr:`492`): Keyword examples updated to be more complete
```

Comment:
I guess this has to be updated with the latest _master_ since we released **14.1.1** meanwhile. (and the diff isn't under the **Upcoming** section anymore I guess)

Reply:
Updated with the latest master

Reply:
Updated to latest version of master